src/vis/vis.js

```diff
@@ -435,15 +427,23 @@ var Vis = View.extend({
    * Only expected to be called if {skipMapInstantiation} flag is set to true when vis is created.
    */
   instantiateMap: function () {
-    this._instantiateMap();
-    this.mapView.invalidateSize();
-  },
-
-  _instantiateMap: function () {
     this._dataviewsCollection.on('add reset remove', _.debounce(this._invalidateSizeOnDataviewsChanges, 10), this);
     this.map.instantiateMap();
   },
 
+  invalidateMapSize: function () {
+    this.mapView.invalidateSize();
```

Comment:
It has to reset to the origin position. After a talk, we are going to rename the function to something like `centerMapToOrigin`, because not only invalidates, it centers the map to the initial bounds/center.